components/alert/alert-toast.js

```diff
@@ -232,13 +232,14 @@ class AlertToast extends LitElement {
 			'd2l-alert-toast-container-lowest': !this._totalSiblingHeightBelow,
 			'vdiff-target': true
 		};
+
 		return html`
 			<div
 				class="${classMap(containerClasses)}"
 				data-state="${this._state}"
 				style="${styleMap(containerStyles)}"
 				@transitionend=${this._onTransitionEnd}>
-				<d2l-alert
+				<d2l-alert role="${ this._state !== states.CLOSED ? 'alert' : '' }"
```

Comment:
This has the added niceness of being in `render()`. 🎉 

Should this used `ifDefined()` and `undefined` so that attribute is fully removed like it was before?